packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -339,7 +339,6 @@ pc.setRemoteSessionDetails = (peerConnection, typeStr, remoteSdp, meetingId) =>
 pc.createOffer = (peerConnection, {meetingId, remoteQualityLevel}) => {
   LoggerProxy.logger.log('PeerConnectionManager->createOffer#createOffer: creating a new offer');
 
-  try {
```

Comment:
The try/catch doesn't return a promise
The promise itself does have a catch block to return an error anyway